src/components/AssetBreadcrumb/stories/AssetBreadcrumb.stories.tsx

```diff
@@ -30,6 +31,15 @@ class CogniteClient extends MockCogniteClient {
   };
 }
 
+const retriewAssets = async (internalId: InternalId): Promise<Asset[]> => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      const result = ASSET_LIST_CHILD.find(asset => asset.id === internalId.id);
+      resolve(result ? [result] : []);
+    }, 100);
```

Comment:
I think it's just enough to return one hardcoded asset here. like `resolve([ASSET_LIST_CHILD[something]])`

Reply:
here we have parent-child relation between assets and important that story renders representable data (instead of single asset)

Reply:
It is required Ivan because of this method used several times. So I can't just hardcode.

Reply:
ah, yep, ok